code/modules/reagents/reagent_containers/food/snacks/meat.dm

```diff
@@ -55,6 +55,21 @@
 	name = "corgi meat"
 	desc = "Tastes like... well, you know."
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/pig
+	name = "meat"
+	desc = "A slab of meat."
+	icon_state = "meat"
+	health = 180
+	filling_color = "#FF1C1C"
+	center_of_mass = list("x"=16, "y"=14)
+	cooked_icon = "meatstake"

```

Comment:
Not going to check. but are you sure this isn't supposed to be "meatsteak"?

Reply:
yes